### DISCUSSION TITLE
Why I need to Hide my Activities and Contributions in an Open-Source Community like GitHub?

### DISCUSSION THREAD

**Discussion by GOOGLE-M** [community/community#31028] [Other Feature Feedback, Questions, & Ideas]
I have a question about the new feature "Make profile private and hide activity".

As GitHub already provides private repo, I do not understand why I still need to hide my activities and contributions in an open-source community?

**Comment by jfullstackdev** [comment]
this is for those who wants a total privacy because without that, how will a user hide his / her public activities ?

**Reply by liberatricebaizer** [reply]
i don't know but i think that's why they  keep hiding there activities

**Reply by jfullstackdev** [reply]
it's personal preference, when they feel like if they are contributing publicly and still they don't want to be exposed, there is the option
